projects/rucio-s3.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: Rucio-S3-compatible access interface for analysis facilities
+postdate: 2023-05-07
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+skillset:
+  - Python
+  - Docker
+status:
+  - Available
+project:
+  - IRIS-HEP
+experiments:
+  - ATLAS
+program:
+  - IRIS-HEP fellow
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Implement a REANA workflow for dark matter searches at RCFM
```

Comment:
this seems still off

Reply:
oh whoops! Indeed that was a copy-paste gone wrong. Good catch @lukasheinrich.